src/main/java/com/starbucks/controllers/RestControllers.java

```diff
@@ -20,29 +20,33 @@ public RestControllers(OrderService orderService) {
 
     @PostMapping()
     public String saveOrder(@RequestBody Order order) {
-        orderService.saveOrder ( order );
-        return order.toString ( );
+        orderService.saveOrder(order);
+        return order.toString();
     }
 
     @PutMapping("/{id}")
-    public String updateOrder(@RequestBody Order order , @PathVariable("id") UUID id) {
-
+    public String updateOrder(@RequestBody Order order, @PathVariable("id") UUID id) {
         try {
-            orderService.existOrder ( id );
+            orderService.existOrder(id);
         } catch (RuntimeException ex) {
-            throw new ResourceAccessException ( "Order does not exist " );
+            throw new ResourceAccessException("Order does not exist");
         }
-        orderService.updateOrder ( order , id );
-        return order.toString ( );
+        orderService.updateOrder(id, order);
+        return order.toString();
     }
 
     @DeleteMapping("/{id}")
     public void deleteOrder(@PathVariable("id") UUID id) {
-        orderService.deleteOrdersById ( id );
+        orderService.deleteOrdersById(id);
     }
 
-    @PatchMapping("/{id}")
-    public void patchOrder(@RequestBody Status status , @PathVariable("id") UUID id) {
-        orderService.patchOrder ( id , status );
+    @PatchMapping("/{status}/{id}")
+    public void patchOrder(@PathVariable("status") Status status, @PathVariable("id") UUID id) {
```

Comment:
it should be @RequestParam not @PathVariable for Status. And you shouldn't have 2 or more path variables in a path without any resource identification like "orders/{id}/status/{status}"